inginious/frontend/plugins/contact_page/pages/api/contact_page.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of UNCode. See the LICENSE and the COPYRIGHTS files for
+# more information about the licensing of this file.
+
+""" Contact page """
+
+import web
+import requests
+import json
+from inginious.frontend.pages.utils import INGIniousPage
+from inginious.frontend.plugins.contact_page.pages.api.slack_url_error import SlackURLError
+
+subject_text = {
+    "subject-comment": "Comentario o reporte de un problema",
+    "subject-new-course": "Peticion para crear un nuevo curso",
+}
+URL_channel = {
+    "subject-comment": "",
+    "subject-new-course": "",
+}
+subject_new_course_id = "subject-new-course"
```

Comment:
This is a local variable, thus it must be within a function. Otherwise, it looks like this variable is used for other modules or may be modified somewhere else, like a global variable.